sources/api/host-containers/src/main.rs

```diff
@@ -105,7 +105,11 @@ mod error {
         #[snafu(display("Logger setup error: {}", source))]
```

Comment:
Typo in commit message: `host-containers: addresssed fmt checks` - maybe `host-containers: cargo fmt`?